src/main/java/seedu/address/logic/commands/ClearConfirmationCommand.java

```diff
@@ -0,0 +1,18 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.Model;
+import seedu.address.ui.CommandDetailChange;
+import seedu.address.ui.CommandTabChange;
+
+/**
+ * Requests confirmation before clear operation.
+ */
+public class ClearConfirmationCommand extends Command {
+
+    public static final String MESSAGE_SUCCESS = "Are you sure you want to clear all data? (Y/N)";
```

Comment:
Maybe we can rename this to sth else like CONFIRMATION_MESSAGE to make it clearer?